<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreezeConfigurationUI</class>
 <widget class="QWidget" name="BreezeConfigurationUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>418</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tit&amp;le alignment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>titleAlignment</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>B&amp;utton size:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>buttonSize</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="buttonSize">
         <item>
          <property name="text">
           <string>Tiny</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">Small</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">Medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">Large</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">Very Large</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="titleAlignment">
         <item>
          <property name="text">
           <string>Left</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Center</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Center (Full Width)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="drawSizeGrip">
         <property name="text">
          <string>Add handle to resize windows with no border</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QCheckBox" name="drawTitleBarSeparator">
         <property name="text">
          <string>Draw separator between Title Bar and Window</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="drawBorderOnMaximizedWindows">
         <property name="text">
          <string>Allow resizing maximized windows from window edges</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QCheckBox" name="outlineCloseButton">
         <property name="text">
          <string>Draw a circle around close button</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QCheckBox" name="drawBackgroundGradient">
         <property name="text">
          <string>Draw window background gradient</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Animations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QLabel" name="animationsDurationLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Anima&amp;tions duration:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>animationsDuration</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="animationsDuration">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string> ms</string>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QCheckBox" name="animationsEnabled">
         <property name="text">
          <string>Enable animations</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Shadows</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Si&amp;ze:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>shadowSize</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="shadowSize">
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">Small</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">Medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">Large</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string comment="@item:inlistbox Button size:">Very Large</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string comment="strength of the shadow (from transparent to opaque)">S&amp;trength:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>shadowStrength</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="shadowStrength">
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Color:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KColorButton" name="shadowColor"/>
       </item>
       <item row="3" column="0" colspan="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Window-Specific Overrides</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="Breeze::ExceptionListWidget" name="exceptions" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>Breeze::ExceptionListWidget</class>
   <extends>QWidget</extends>
   <header>config/breezeexceptionlistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>titleAlignment</tabstop>
  <tabstop>buttonSize</tabstop>
  <tabstop>outlineCloseButton</tabstop>
  <tabstop>drawBorderOnMaximizedWindows</tabstop>
  <tabstop>drawBackgroundGradient</tabstop>
  <tabstop>drawSizeGrip</tabstop>
  <tabstop>drawTitleBarSeparator</tabstop>
  <tabstop>animationsEnabled</tabstop>
  <tabstop>animationsDuration</tabstop>
  <tabstop>shadowSize</tabstop>
  <tabstop>shadowStrength</tabstop>
  <tabstop>shadowColor</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>animationsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>animationsDurationLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animationsEnabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>animationsDuration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
